corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html

```diff
@@ -72,7 +72,8 @@
   </p>
   <p class="help-block">
     {% blocktrans %}
-      This feature allows you to quickly discover the cases of your interest.<br />
+      This feature allows you to quickly explore your case data for ad-hoc data
+      queries or to identify unclean data.<br />
       This is a Preview Feature.
```

Comment:
nit: elsewhere we call these "Feature Preview"s

Reply:
haha. yep. updated--thanks!

![whoops](https://media.giphy.com/media/M3FQyvutliSgE/giphy.gif)